src/scheduler/workers/unpublish/findNonRespondentsForList.ts

```diff
@@ -0,0 +1,68 @@
+import { prisma } from "server/models/db/prisma-client";
+import { ListJsonData } from "server/models/types";
+import { schedulerLogger } from "scheduler/logger";
+import { List, Prisma } from "@prisma/client";
+import { findReminderToSend } from "./dayBefore/findReminderToSend";
+import {Meta} from "./types";
+
+export async function findNonRespondentsForList(list: List, meta: Meta) {
+  return await getLists(list, meta, false);
+}
+
+export async function findNonRespondentsForListToUnpublish(list: List, meta: Meta) {
+  return await getLists(list, meta, false);
+}
```

Comment:
Out of curiosity what is the purpose of these two functions. The only difference it seems is their name.